homeassistant/components/tailwind/cover.py

```diff
@@ -77,6 +78,14 @@ async def async_open_cover(self, **kwargs: Any) -> None:
                 translation_domain=DOMAIN,
                 translation_key="door_locked_out",
             ) from exc
+        except TailwindDoorAlreadyInStateError:
+            LOGGER.debug(
```

Comment:
We do not have to create an `HomeAssistant` error here; as this is a debug log and the error isn't raised.